jekyll/_cci2/version-control-system-integration-overview.adoc

```diff
@@ -98,6 +107,11 @@ include::../_includes/partials/tips/check-github-type.adoc[Check your GitHub int
 
 ^2^ _Enable by using the GitHub App integration alongside the OAuth app integration. More details xref:github-integration#using-github-app-functionality[here]._
 
+^3^ _When Github Checks are enabled we automatically disable Github status updates to avoid duplication of statuses. You can manage GitHub status updates via project settings._
```

Comment:
```suggestion
^3^ _When GitHub Checks are enabled we automatically disable GitHub status updates to avoid duplication of statuses. You can manage GitHub status updates via project settings._
```